developer_resources/DCV_with_ParallelCluster.md

```diff
@@ -0,0 +1,457 @@
+<span style="display: inline-block;">
+
+# GUI FPGA Development Environment with NICE DCV and ParallelCluster
+
+Deploy a CloudFormation template to Launch an EC2 instance with the FPGA Developer AMI that has DCV Remote Desktop and ParallelCluster.
+
+## Table of Contents
+
+  * [Overview](#overview)
+  * [Requirements](#requirements)
+  * [Architecture](#architecture)
+  * [Cost](#cost)
+  * [Duration](#duration)
+  * [Step-by-step Guide](#step-by-step-guide)
+     * [Subscribe to AWS FPGA Developer AMI](#subscribe-to-aws-fpga-developer-ami)
+     * [Launch with CloudFormation](#launch-with-cloudformation)
+     * [Connect to the DCV Remote Desktop session](#connect-to-the-dcv-remote-desktop-session)
+     * [Launch Vivado](#launch-vivado)
+     * [ParallelCluster Configuration](#pcluster-config)
+     * [Building a DCP On ParallelCluster Using SGE](#building-a-dcp-on-parallelcluster-using-sge)
+     * [Building a DCP On ParallelCluster Using Slurm](#building-a-dcp-on-parallelcluster-using-slurm)
```

Comment:
It's cool that they added docs on how to use AWS ParallelCluster for FPGA development.  Using AWS ParallelCluster to add support to FireSim manager for running with a cluster managing the build and runtime node provisioning and dolling them out might be a way to develop support before asking IT to make them available in a cluster in the datacenter.

Reply:
Admittedly i don't know a thing about ParallelCluster, i'll have to look into it.